src/lib/Actions.js

```diff
@@ -0,0 +1,32 @@
+import * as Api from "./ApiHelper";
+
+function increaseABit(number) {
+    return parseInt(1.2 * number);
+}
+
+function getTestProvider (web3) {
+    return new web3.providers.WebsocketProvider('ws://localhost:8545')
+}
+
+async function mineBlock (web3) {
```

Comment:
mine block is not something that could work on testnet. you should not use it. it is only for internal tests.